department-of-veterans-affairs/caseflow
# Fixing a bunch of frontend sentry errors

### Description
Fixes the following sentry trivial errors (most are just adding null checks):
https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/1961/
https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/1958/
https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/1977/
https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/1976/
https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/2005/

Fixes an error that occurs when a user creates a tag and then immediately deletes it. When this happens, our delete code cannot find the tag and we throw an error. This is a partial fix, this removes the tag locally, but when we receive a successful response from the server we'll add the tag back.
https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/1984/

We're adding logging to identify this error:
https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/1960/

### Testing Plan
1. https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/1976/:
   1. Go into document, add a comment with `-)`
   1. Go to list page, and search for `-)`them.
1. https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/2005/:
   1. On PDF view, hit `alt + enter`
1. https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/1961/
   1. Quickly flip between pdf docs and hit page down (or `fn + down arrow`)
1. https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/1958/
   1. Unsure how to test, just added a null check
1. https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/1977/
   1. Unsure how to test, just added a null check
1. https://sentry.ds.va.gov/department-of-veterans-affairs/caseflow/issues/1984/events/latest/
   1. Load pdf document page
   1. In chrome dev tools change network speed to slow 3G
   1. Add and then immediately delete a tag
